seal_rookery/upload_images.py

```diff
@@ -59,30 +59,24 @@ def resize_image(original: str, size: str) -> str:
     if not os.path.exists(os.path.dirname(new_filepath)):
         os.mkdir(os.path.dirname(new_filepath))
 
-    svg = True if "svg" in original else False
-    if svg:
+    if "svg" in original:
         image = pyvips.Image.thumbnail(original, int(size), height=int(size))
         new_filepath = new_filepath.replace(".svg", ".png")
         image.write_to_file(new_filepath)
     else:
-        with open(original, "r+b") as f:
-            with Image.open(f) as image:
-                width, height = image.size
-                if height < int(size):
-                    ratio_height = (int(size) / width) * height
-                    cover = resizeimage.resize_cover(
-                        image, [int(size), ratio_height], validate=False
-                    )
+        with open(original, "r+b") as f, Image.open(f) as image:
```

Comment:
```
seal_rookery/upload_images.py:68:9: SIM117 Use a single `with` statement with multiple contexts instead of nested `with` statements
   |
66 |           image.write_to_file(new_filepath)
67 |       else:
68 | /         with open(original, "r+b") as f:
69 | |             with Image.open(f) as image:
   | |________________________________________^ SIM117
70 |                   width, height = image.size
71 |                   if height < int(size):
   |
   = help: Combine `with` statements
```

https://docs.astral.sh/ruff/rules/multiple-with-statements/